docker/.env.example

```diff
@@ -0,0 +1,38 @@
+MAINNET_RPC_URL="https://rpc.cheqd.net:443"
+TESTNET_RPC_URL="https://rpc.cheqd.network:443"
+RESOLVER_URL="https://resolver.cheqd.net/1.0/identifiers/"
+APPLICATION_BASE_URL="http://localhost:8787"
+ALLOWED_ORIGINS="http://localhost:8787"
+
+# Database
+ENABLE_EXTERNAL_DB="string,default:false"
+EXTERNAL_DB_CONNECTION_URL="postgres://<user>:<password>@<host>:<port>/<database>"
+EXTERNAL_DB_ENCRYPTION_KEY="<db-encryption-key>"
+EXTERNAL_DB_CERT="<path/to/extenal-db-cert>"
+POSTGRES_USER="<user>"
```

Comment:
Is it possible to pass into docker container DB_URL instead of several vars with username/password?

Reply:
No, It isn't.